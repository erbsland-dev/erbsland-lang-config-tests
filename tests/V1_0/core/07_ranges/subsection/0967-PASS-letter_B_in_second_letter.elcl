# Erbsland Configuration Language Test File
[main.aBname]