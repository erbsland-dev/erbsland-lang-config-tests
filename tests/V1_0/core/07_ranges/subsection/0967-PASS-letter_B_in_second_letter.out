main = IntermediateSection()
main.abname = SectionWithNames()
